submissions/Zakkarat/html-forms/stylesheet/language-choose.css

```diff
@@ -83,3 +83,9 @@
   font-size: 20px;
   cursor: pointer;
 }
+.no-button-style {
+  -webkit-appearance: none;
+  appearance: none;
+  background-color: white;
+  border: 0px;
+}
```

Comment:
There is no empty line at the end of the file. This may be a silly remark, but please try not to do that. Just read why it's important. Please install Prettier.